README.md

```diff
@@ -178,13 +178,14 @@ function MyComponent() {
 Providing the default English value in the code makes it possible to use it as a fallback value when strings are not yet translated and to extract the strings from the source code to the translations YAML file.
 
 #### Plurals 
-When using [plurals](https://www.i18next.com/translation-function/plurals), you need to provide default values for at least the `one` and `other` forms. You can also pass a default value for the `zero` form, if you want it to be different from the `other` form. This can be done by passing the default values in the [options](https://www.i18next.com/translation-function/essentials#overview-options) of the `t` function.
+When using [plurals](https://www.i18next.com/translation-function/plurals), you need to provide default values for the `zero`, `one`, `two`, `few`, `many` and `other` forms, as some languages have specific forms for all of these. These forms should be provided in the [translations YAML](./src/modules/service-catalog/translations/en-us.yml) file. This can be done manually or by running the extract script, namely [extract-strings.mjs](./bin/extract-strings.mjs), after adding the string in the code.
```

Comment:
Nit: 
- I would not link the service catalog translation file specifically since this thing is valid for all translation files,  or at least adding a `for example`
- I would just refer to the next section for the extract script, since it has a more detailed explanation